blueprints/addon/files/.github/workflows/ci.yml

```diff
@@ -0,0 +1,66 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - [main, master]
+  pull_request: {}
+
+jobs:
+
+  lint:
+    name: "Lint"
+    env:
+      CI: true
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install node
+      uses: actions/setup-node@v2-beta
```

Comment:
Afaik this has been stable for a while ( `actions/setup-node@v2`)

Reply:
Thank you!! Fixed.